pom.xml

```diff
@@ -40,6 +40,7 @@
 <!--        <bnd-maven-plugin.version>3.4.0</bnd-maven-plugin.version>-->
 <!--        <bnd-baseline-maven-plugin.version>4.2.0</bnd-baseline-maven-plugin.version>-->
 <!--        <build-helper-maven-plugin.version>1.12</build-helper-maven-plugin.version>-->
+        <maven.compiler.release>17</maven.compiler.release>
```

Comment:
Well, this is totally unacceptable :) MP is only JDK 11 minimum, and we should not go over that.

Reply:
Do you mean I should update this to 11 instead? I can do that

Reply:
No, remove it altogether, this is inherited from the parent.